frontend/src/components/AnalyticsTable/AnalyticsTable.tsx

```diff
@@ -208,6 +300,9 @@ const Table = ({ type }: TableProps) => {
   }, [type, drivers.length]);
 
   return (
+    <>
+    <DateFilter/>
+    <AnalyticsOverview stats={generateStatsArr()} />
```

Comment:
move these to the analytics page in Admin. when connecting to backend, the analytics page will handle getting stats and passing the table data to all components, including the table and analytics overview